cypress/e2e/reports/activity-log/activity-log.spec.js

```diff
@@ -0,0 +1,232 @@
+          .add(1, "days")
+          .format("YYYY-MM-DD")}`
+      );
+    });
+
+    it("Does not set default date url parameters if they are set", () => {
+      cy.visit("/reports/activity-log/resources?from=2020-07-14&to=2020-07-16");
+      cy.location("search").should(
+        "not.eq",
+        `?from=${moment().add(-6, "days").format("YYYY-MM-DD")}&to=${moment()
+          .add(1, "days")
+          .format("YYYY-MM-DD")}`
+      );
+    });
+
+    it("Displays all url parameters on filters", () => {
+      cy.waitOn("/v1/env/audits*", () => {
+        cy.visit(
+          "/reports/activity-log/resources?from=2022-07-14&to=2022-07-29&resourceType=content&actionByUserZUID=5-84d1e6d4ae-s3m974&sortBy=happenedAt"
+        );
+      });
+      cy.get('[data-cy="filters"]').within(() => {
+        cy.contains("Sort By").next().contains("Oldest First");
+        cy.contains("Resource Type").next().contains("Content");
+        cy.contains("User").next().contains("Andres Galindo");
+      });
+    });
+
+    it("Adds and modifies url parameters via filters", () => {
+      cy.waitOn("/v1/env/audits*", () => {
+        cy.visit(
+          "/reports/activity-log/resources?from=2022-07-14&to=2022-07-16"
+        );
+      });
+      cy.get('[data-cy="filters"]').contains("Sort By").next().click();
+      cy.get('li[data-value="happenedAt"]').click();
+
+      cy.get('[data-cy="filters"]').contains("Resource Type").next().click();
+      cy.get('li[data-value="content"]').click();
+
+      cy.get('[data-cy="filters"]').contains("User").next().click();
+      cy.get('li[data-value="5-84d1e6d4ae-s3m974"]').click();
+
+      cy.get('[data-cy="filters"]').contains("From").next().click();
+      cy.get(
+        '.MuiDateRangePickerDay-root button[aria-label="Jul 18, 2022"]'
+      ).click();
+      cy.get('[data-cy="filters"]').contains("To").next().click();
+      cy.get(
+        '.MuiDateRangePickerDay-root button[aria-label="Jul 30, 2022"]'
+      ).click();
+      cy.location("search").should(
+        "eq",
+        "?from=2022-07-18&sortBy=happenedAt&resourceType=content&actionByUserZUID=5-84d1e6d4ae-s3m974&to=2022-07-30"
+      );
+    });
+  });
+
+  describe("Resource Tab", () => {
+    it("Navigates to Resource Detail on Resource Item click", () => {
+      cy.waitOn("/v1/env/audits*", () => {
+        cy.visit(
+          "/reports/activity-log/resources?from=2022-07-14&to=2022-07-16"
+        );
+      });
+      cy.wait(1000);
+      cy.get(".MuiListItem-root").contains("new all fields").click();
+      cy.location("pathname").should(
+        "eq",
+        "/reports/activity-log/resources/7-f28fd4d4a9-qtjb66"
+      );
+    });
+  });
+
+  describe("Skeletons", () => {
+    it("Displays All Skeletons while initial API is called", () => {
+      cy.intercept("/v1/env/audits*", (req) => {
+        req.continue(async (res) => {
+          await new Promise((resolve) => setTimeout(resolve, 4000));
```

Comment:
This is subtle complex. Please add a comment explaining. My understanding is we are inserting an explicit 4 seconds of time to ensure skeletons are rendered to test for their presence.